interview-questions.md

```diff
@@ -8,36 +8,117 @@ Then, research the question to expand on your answer. Even if you feel you have
+  
+  def index
+    @blogs = Blog.all
+  end
+
+
+action: "POST" location: /students :The create controller method, this method is responsible for adding data to the table in the database if the data is valid.
+   
+   def create
+        @blog = Blog.create(blog_params)
+        if @blog.valid?
+        redirect_to blogs_path
+        end
+    end
+
+action: "GET" location: /students/new :The new controller method the action would be to create a new model instance, new is a restful route that displays a form to the user.
+
+     def new
+        @blog = Blog.new
+    end
+
+action: "GET" location: /students/2 The show controller method allows the user to display that particular column from the database, show is a restful route that looks for one item by the id.
+
+    def show
+        @blog = Blog.find(params[:id])
+    end
+
 
-action: "POST" location: /students
+action: "GET" location: /students/2/edit  :The edit controller method  the action is responsible for modifying the item in the database.
 
-action: "GET" location: /students/new
+    def edit 
+        @blog = Blog.find(params[:id])
+    end
 
-action: "GET" location: /students/2
 
-action: "GET" location: /students/2/edit
+action: "PATCH" location: /students/2 The update controller method allows the user to update a column in the table and modify it.
 
-action: "PATCH" location: /students/2
+     def update
+       @blog = Blog.find(params[:id])
+       blog.update(blog_params)
+    end
 
-action: "DELETE" location: /students/2
+action: "DELETE" location: /students/2 :The destroy controller method is repsonsible for deleting a certain column in the table.
+
+    def destroy
+        @blog = Blog.find(params[:id])
+        if @blog.destroy
+        redirect_to blogs_path
+        end
+    end
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+1. The story needs a "defintion of completion" for the to do list. The list must define what needs to be done so the user knows when the list is completed.
+
+2. As a user I can add items to do into the a form so it can be added to the list, each item will be listed as how important it is to complete.
+
+3. So on the list, the time for completion will also be listed for each item that is considered a reasonable time frame for that particlar task.
+
+4. As a user, the items listed will only be added for the week of to prevent confusion or the feeling of too much to do.
+
+5. As a user I want to manage the list accordingly so as a user I can update/edit any of my items in my todo's list just in case it's not comepleted.
+
+6. As a user I can delete any of my items on my todo's list if anything changes and that event is canceled.
```

Comment:
I really like this format you used a couple of times where you say As a user, I can do xyz.